wellen/src/wavemem.rs

```diff
@@ -800,6 +786,19 @@ impl SignalEncoder {
                             });
                     self.max_states = States::join(self.max_states, states);
                 } else {
+                    let value_bits: &[u8] = match value[0] {
+                        b'b' | b'B' => &value[1..],
+                        _ => value,
```

Comment:
I wonder if this is ever triggered now? It can be replaces with 
``` rust
let value_bits: &[u8] =  &value[1..];
```
and the tests still pass.